.github/workflows/release.yml

```diff
@@ -0,0 +1,31 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - '*'
```

Comment:
Yes, it will prevent unwanted tags from triggering CI errors.
I pushed your suggestion [here](https://github.com/opentofu/setup-opentofu/pull/10/commits/d29e25889fab0a003a749af22b604b7b95f82e0d)

Reply:
I changed it to `- 'v[0-9]+.[0-9]+.[0-9]+*'` because I think the idea was to mach perfectly the tag format we want to use. WDYT ?